client/app/account/billing/confirmTerminate/billing-confirmTerminate.html

```diff
@@ -104,6 +104,23 @@ <h1 data-translate="terminate_service_title"></h1>
                     </label>
                 </div>
 
+                <h5 class="oui-header_5"
+                    data-translate="terminate_service_next-steps_title"></h5>
+
+                <div class="oui-radio"
+                     data-ng-repeat="futureUse in TerminateServiceCtrl.futureUses">
+                    <input class="oui-radio__input"
+                           type="radio"
+                           name="futureUses"
+                           data-ng-click="TerminateServiceCtrl.futureUse = futureUse"
+                           data-ng-value="futureUse"
+                           id="futureUse-{{futureUse}}" />
+                    <label class="oui-radio__label-container" for="futureUse-{{futureUse}}">
```

Comment:
You could enhance performances by having one-time binding: `for="futureUse-{{:: futureUse}}`